src/components/Header.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ReactComponent as Logo } from "../images/sun.svg";
+
+export default function Header(props) {
+  const {
+    view,
+    unit,
+    currentWeather,
+    handleUnit,
+    handleChange,
+    handleClick,
+    handleView,
+    handleClear
+  } = props
+
+  return (
+    <header>
+      <h1><Logo />Weather</h1>
+      <div>
+        <input required type="text" name="cityName" onChange={handleChange} onClick={handleClear} placeholder="Enter City... " />
+        <button onClick={handleClick}>Get Weather</button>
```

Comment:
What are your thoughts on using the components with click handlers vs using a `<form>` submission flow?

Reply:
Normally, I would wrap everything in a form and use `onSubmit`. I almost did that here, again; however, since it's only one input and a simple `get` request, it just didn't feel necessary. That said, when I was done with the app, I kind of regretted this, as I realized it would have been nice if you could click the button OR hit `enter`, and an `onSubmit` would have made that easier. If this was a real-life project, I probably would have updated it.